uint8 mission_type
uint8 target_waypoint
uint8 current_status
uint8 error_code
